Pcap++/src/DpdkDeviceList.cpp

```diff
@@ -400,6 +400,54 @@ bool DpdkDeviceList::startDpdkWorkerThreads(CoreMask coreMask, std::vector<DpdkW
 	return true;
 }
 
+bool DpdkDeviceList::startDpdkWorkerThread(CoreMask coreMask, DpdkWorkerThread* worker)
```

Comment:
Please add a test in https://github.com/seladb/PcapPlusPlus/blob/master/Tests/Pcap%2B%2BTest/Tests/DpdkTests.cpp to test this new method